### DISCUSSION TITLE
Add keyboard navigation on photos and album pages

### DISCUSSION THREAD

**Discussion by mstewart14** [immich-app/immich#6202] [Feature Request]
it's nice having the keyboard control when viewing images directly.  It would be nice having keyboard control of navigation when viewing the photos page itself or an album.  Page Up, Page Down, Up and Down arrows would be nice defaults.  

- [x] Web
- [ ] Server
- [ ] Mobile

**Comment by kikendo** [comment]
Immich is already receiving and acting on keyboard input, it's just responding wrong. 
Keyboard navigation is the most basic of navigations, we don't all live glued to a smartphone or touchscreen, how is this "not planned"?

**Reply by bo0tzz** [reply]
It's not "not planned", github just doesn't have "duplicate" as a specific close reason for issues.

**Reply by kikendo** [reply]
but what I posted isn't a duplicate. It's not working and that's what I reported.
This feature exists, that's why you just marked it as "solved". And it isn't working properly.

Please reopen my issue.

**Comment by Snowknight26** [comment]
I think this functionality is effectively implemented with https://github.com/immich-app/immich/pull/9448, unless I'm misunderstanding something.

> Currently only PageDown/PageUp trigger the focus, to match the product that shall not be named, but once focus is given to the asset grid, arrow keys and home/end also work as expected.

**Reply by kikendo** [reply]
II came here because I was told I posted a "duplicate issue" but I was not. This feature isn't working properly.
I want #9947 reopened @bo0tzz please.

**Reply by Snowknight26** [reply]
A fix has been implemented and the code is available, but if you don't manually build Immich yourself, you need to wait for the next version to be released, which is presumably why it's closed.

**Reply by bo0tzz** [reply]
That's correct, the PR was only merged last week and there hasn't been a release since then.

**Reply by kikendo** [reply]
Thanks for answering that, it would be nice to understand why things get closed instead of having them closed without explanation and when asked given confusing reasonings. I'm not contributing on the code of course but I am trying to help that is all.

No rush at all on this as a user, just wanted to report my findings. Glad to hear it has been fixed
